build.sh

```diff
@@ -17,7 +17,7 @@ function set_gemfile(){
 echo "Setup Spree defaults and creating test application..."
 rm_gemfile_lock
 bundle check || bundle install
-bundle exec rake test_app
+DB=postgres bundle exec rake test_app
```

Comment:
This is an environment variable we can set in Semaphore. Semaphore allows to set global environment variables. Alternatively, we can change the build script to include it:

```sh
DB=postgres bash build.sh
```

So we don't have to change the code for this.